File: rmt/src/test/scala/de/flower/rmt/ui/manager/page/events/EventsPageTest.scala
package de.flower.rmt.ui.manager.page.events

import de.flower.rmt.test.AbstractWicketIntegrationTests
import org.testng.annotations.Test
/**
 * 
 * @author flowerrrr
 */

class EventsPageTest extends AbstractWicketIntegrationTests {

    @Test
    def testRender() {
        wicketTester.startPage(new EventsPage())
        wicketTester.dumpPage()
    }


}

File: rmt/src/test/java/de/flower/rmt/ui/player/page/events/EventsPageTest.java
package de.flower.rmt.ui.player.page.events;

import de.flower.rmt.test.AbstractWicketIntegrationTests;
import org.testng.annotations.Test;

/**
 * @author flowerrrr
 */

public class EventsPageTest extends AbstractWicketIntegrationTests {

    @Test
    public void testRender() {
        wicketTester.startPage(new EventsPage());
        wicketTester.dumpPage();
    }
}